docs/acp/user/app-manifest.md

```diff
@@ -95,7 +95,7 @@ For the manifest, make sure to follow these conditions:
 |assets   |An array of objects represented as application assets. Each object has the following keys:<ul><li>type—type of the asset. Possible values:<ul><li>icon—displayed on the application tile and on top of the application detail page.</li><li>image—displayed in a carousel on the application detail page.</li><li>video—displayed in a carousel on the application detail page. Allows only videos hosted on https://wistia.com.</li></ul><li>url—a relative path to the asset. Possible extensions:<ul><li>jpeg</li><li>png</li><li>svg</li><li>url to a video hosted on https://wistia.com</li></ul></ul>   |See *assets example* under this table.   |
 |labels   |An array of strings. Displays label icons on the application detail page according to the label. Possible values:<ul><li>Silver Partner</li><li>Gold Partner</li><li>New</li><li>Popular</li><li>Free Trial</li></ul>   |See *labels example* under this table.   |
-|resources   |An array of objects represented as application resources (links, documents). The resource object has the following keys:<ul><li>title—name of the resource</li><li>url—full URL to the resource</li>type—type of the resource. Optional. The type affects a displayed icon. Possible values:<ul><li>internal-documentation</li><li>homepage</li><li>user-documentation</li><li>developer-documentation</li><li>release-notes</li></ul><li>fileType—type of the resource file (in case if the url leads to a file and if `type=internal-documentation`). Optional. The files are displayed directly in the AppCatalog, without redirecting to another page. Possible values:<ul><li>pdf</li><li>markdown</li></ul></ul>   |See *resources example* under this table.   |
+|resources   |An array of objects represented as application resources (links, documents). The resource object has the following keys:<ul><li>title—name of the resource</li><li>url—full URL to the resource</li>type—type of the resource. Optional. The type affects a displayed icon. Possible values:<ul><li>internal-documentation</li><li>homepage</li><li>user-documentation</li><li>developer-documentation</li><li>release-notes</li></ul><li>fileType—type of the resource file (in case if the url leads to a file and if `type=internal-documentation`). Optional. The files are displayed directly in the AppCatalog, without redirecting to another page. Possible values:<ul><li>pdf</li><li>markdown</li><li>spryker-docs</li></ul></ul>   |See *resources example* under this table.  {% info_block infoBox "Info" %} the `"fileType": "markdown"` is still supported, however, it is now recommended to use `spryker-docs` instead. The `url` will be a full url to https://docs.spryker.com document. {% endinfo_block %} |
```

Comment:
```suggestion
|resources   |An array of objects represented as application resources (links, documents). The resource object has the following keys:<ul><li>title—name of the resource</li><li>url—full URL to the resource</li>type—type of the resource. Optional. The type affects a displayed icon. Possible values:<ul><li>internal-documentation</li><li>homepage</li><li>user-documentation</li><li>developer-documentation</li><li>release-notes</li></ul><li>fileType—type of the resource file (in case if the url leads to a file and if `type=internal-documentation`). Optional. The files are displayed directly in the AppCatalog, without redirecting to another page. Possible values:<ul><li>pdf</li><li>markdown</li><li>spryker-docs</li></ul></ul>   | See the *resources example* under this table.  The `"fileType": "markdown"` is still supported; however, it is now recommended to use `spryker-docs` instead. The `url` is a full URL to [Spryker documentation](https://docs.spryker.com).  |
```